src/services/web3/web3.plugin.ts

```diff
@@ -67,6 +67,11 @@ type PluginState = {
 };
 
 async function isSanctionedAddress(address: string): Promise<boolean | null> {
+  if (
+    !process.env.VUE_APP_WALLET_SCREENING ||
+    process.env.VUE_APP_WALLET_SCREENING !== 'true'
+  )
```

Comment:
Could you move add this ENV var to config.service.ts `env` attribute and resolve the variable to a boolean type as expected. So in the config service something like:
```ts
WALLET_SCREENING: (process.env.VUE_APP_WALLET_SCREENING && process.env.VUE_APP_WALLET_SCREENING === 'true') || false
```
Then above you should just be able to do:
```suggestion
  if (configService.env.WALLET_SCREENING)
```

Reply:
Updated